11-model-comparison/11_model_comparison.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # You won't have to calculate this stuff yourself. Turing will do that for you!
</pre><p>Mind you don't sound too patronizing to the audience, but also, the ideas here don't quite flow. Bayes theorem isn't that hard to calculate--the confusing part is, why are you introducing it at all, at <em>this </em>stage? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>